infer/tests/codetoanalyze/java/pulse/taint/Streams.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package codetoanalyze.java.pulse;
+
+import java.io.InputStream;
+
+import codetoanalyze.java.pulse.InferTaint;
+
+import java.io.IOException;
+import java.io.File;
+
+public class Streams {
+
+  int bufferSize = 1024;
+
+  void copyBad() throws IOException {
+    InputStream tainted = (InputStream) InferTaint.inferSecretSource();
+    InferTaint.inferSensitiveSink(read(tainted.readAllBytes()));
```

Comment:
Seems like `readAllBytes` is available since java 9 but internally majority of our stuff is still on java 8 so it breaks the tests

Reply:
For this example, I can try whether the issue is reproducible with just `toString` instead of `readAllBytes`. In general Infer's default propagation seems to handle `readAllBytes` correctly - the traditional `read` methods where the first argument should get tainted probably require some more attention.

Reply:
Rebased the branch and replaced `readAllBytes()` by `toString().getBytes()`, which should be equivalent for taint analysis.